main.nf

```diff
@@ -79,24 +79,16 @@ process manta {
     module 'bioinfo-tools'
     module "$params.modules.manta"
 
+    errorStrategy { task.exitStatus == 143 ? 'retry' : 'terminate' }
+    time { params.long_job * 2**task.attempt }
```

Comment:
What's the effect of this? First retry will get double time, second quadruple, etc.?